packages/cms-editor/src/query/query-client.ts

```diff
@@ -1,9 +1,20 @@
+import type { CmsDataArgs, CmsReadArgs } from '@axonivy/cms-editor-protocol';
 import { QueryClient } from '@tanstack/react-query';
+import { useMemo } from 'react';
 
 export const initQueryClient = () => {
   return new QueryClient();
 };
 
+export const useQueryKeys = () => {
+  return useMemo(() => {
+    return {
+      dataKey: (args: CmsDataArgs) => genQueryKey('data', args),
+      readKey: (args: CmsReadArgs) => genQueryKey('read', args)
+    };
+  }, []);
+};

```

Comment:
you can also use the direct return syntax
```suggestion
export const useQueryKeys = () => (
  useMemo(() => ({
    dataKey: (args: CmsDataArgs) => genQueryKey('data', args),
    readKey: (args: CmsReadArgs) => genQueryKey('read', args)
  }), []);
);
```